### PR TITLE
Add base version of app

### LINKED ISSUE #1: Create base version of app
- [x] Main page with link generator
- [x] Allow custom links name
- [x] Use firebase as storage

### PR DESCRIPTION
**Pull request by turbaszek**
closes #1

### CODE REVIEW

File: cmd/tnijto/main.go [hunk 1]
```diff
@@ -1,3 +1,66 @@
...
+	"github.com/gorilla/mux"
+	utils "github.com/turbaszek/tnijto/pkg"
+	"log"
+	"net/http"
+	"time"
+)
+
+var config = utils.NewConfig()
+
+func main() {
```
Comment (janusz-chludzinski) [inline cmd/tnijto/main.go:15]: I'd add some kind of log information to indicate where to look for tnijto when it starts. It took me a while to find port and I didn't notice that app is up 😉

File: cmd/tnijto/main.go [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+	name := r.FormValue("name")
+	generated := fmt.Sprintf("https://%s/%s", config.Hostname, name)
+
+	linkResponse := Link{link, name, generated}
+
+	js, err := json.Marshal(linkResponse)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
```
Comment (janusz-chludzinski) [inline cmd/tnijto/main.go:58]: I'd extract error handling to function or some dedicated component. WDYT?

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module github.com/turbaszek/tnijto
+
+go 1.13
```
Comment (janusz-chludzinski) [inline go.mod:3]: I think we can use 1.14 and perhaps we could go for 1.15 whichc was released august 2020 :)

Comment (turbaszek) [reply go.mod:3]: We can, I just need to install it 😃

File: cmd/tnijto/main.go (hunk 2 above)
Comment (turbaszek) [reply cmd/tnijto/main.go:58]: I thought about it, but not sure if necessary yet

File: cmd/tnijto/main.go (hunk 1 above)
Comment (turbaszek) [reply cmd/tnijto/main.go:15]: I will add. Btw the information is in README.md but... it's not merged yet so not easily visible.

**Comment by turbaszek** [comment]
@janusz-chludzinski @TobKed are we good with moving on?

**Review by TobKed — APPROVED** [review]
LGTM! Good job!

**Pull request merged**